docs/reference-guide/index.md

```diff
@@ -12,6 +12,9 @@ myst:
 This reference guide details application programming interfaces (APIs), functions, modules, and objects included in Plone, describing what they are and what they do.
 Its audience is primarily developers of Plone and its projects and add-ons.
 
+{doc}`Cookieplone make commands </reference-guide/cookieplone-make-commands>`
+: This lists all the `make` commands in a cookieplone project.
```

Comment:
When the title is the same as what you would insert, you can simplify the MyST syntax.

```suggestion
{doc}`/reference-guide/cookieplone-make-commands`
:   This chapter describes all the `make` commands in a {term}`Cookieplone` project.
```